modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js

```diff
@@ -17,244 +17,302 @@
  */
 
 var palette = new Rickshaw.Color.Palette({scheme: "classic9"});
-var graphMap = {};
 
 function drawGraph_connectedcup(from, to) {
+    $("#y_axis-temperature").html("");
```

Comment:
fixed it with https://github.com/wso2/product-iots/pull/190/commits/b8335a0a38f1f77f64dd8cb8d84f6cb4930c5011